### PR TITLE
Updates caption on MN slide

### PR DESCRIPTION
**Pull request by mnnhbc**
TL;DR; This PR requests an updated caption on MN's slide 

This resolves #11 

cc: mnnhbc

### CODE REVIEW

File: _slides/06-mnnhbc.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I'm Matthew.  I like guitar, have a moustashe problem and have arms and legs which look slightly octopoid
```
Comment (gusshawstewart) [inline _slides/06-mnnhbc.md:6]: This is a really nice change! 😄

File: _slides/06-mnnhbc.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (gusshawstewart) [inline _slides/06-mnnhbc.md]: have you thought about doing it like this:
```suggestion
title: "Welcome to the greatest slide deck"
```

**Review by gusshawstewart — APPROVED** [review]
LGTM 🏅 😸

**Review by clarkec21 — APPROVED** [review]
Nice work 🩲

**Pull request merged**